package.json

```diff
@@ -4,7 +4,7 @@
   "description": "--",
   "main": "index.js",
   "scripts": {
-    "start": "node src",
+    "start": "nodemon src",
```

Comment:
You're using nodemon to run your application. However, you have it installed as a devDependencies. And, in your Dockerfile (line 11), you're only installing regular dependencies (not devDependencies). This means that nodemon will not be available to start the application in production. Take a look into that to ensure you're using the correct method to start your application or ensure the dependency is installed correctly.

Reply:
Good catch! Opened a PR to fix this.